conda_smithy/templates/create_conda_build_artifacts.sh.tmpl

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
Why does this file have to be a `.tmpl` file? It doesn't have any jinja2 right?

Reply:
It doesn't (unless we think doing some jinja2 rendering would be useful at some point), but it seemed slightly more natural to me to do it that way since the `store_build_artifacts` setting is keyed to the CI providers. Now that I've done it, I'm not so sure, because of the extra bits of code needed to ensure that the files get deleted on a re-render.

The other way I see is to put it in `feedstock_content/.scripts` and add a bit of code to add the scripts to the `skip_files` list when none of the providers have `store_build_artifacts` set. Do you think that's better?

Reply:
> The other way I see is to put it in feedstock_content/.scripts and add a bit of code to add the scripts to the skip_files list when none of the providers have store_build_artifacts set.

Isn't that what `extra_platform_files` is supposed to do?

Reply:
nvm, that's for CI specific files and not for something that's common. Let's open an issue for this. We don't have to do this in this PR.

Reply:
So does that mean leave as is, and we'll change it in another PR? Or does it mean move the files to `feedstock_content/.scripts`, and we'll implement the proper way to have them conditionally included in another PR?

Reply:
Up to you. I'm fine with either.

Reply:
I'll leave it as-is then, since at least that will not have the scripts present when not needed.